packages/account/src/Sections/Profile/PersonalDetails/verify-button.tsx

```diff
@@ -30,6 +34,14 @@ export const VerifyButton = observer(() => {
         }
     }, [WS.isSuccess, history]);
 
+    useEffect(() => {
+        if (next_otp_request || is_request_button_diabled) {
+            setIsPhoneFieldDisable(true);
```

Comment:
why do we add this useEffect to disable the phone number field in verify button component and not in personal details component

Reply:
I have a PR to remove the useEffect in verify-button, but the PR is not merge yet because of Design is not ready yet: https://github.com/suisin-deriv/deriv-app/pull/103/files#diff-8d277cca4ce6dbec30cb7ab7ae0969b3a713ec50d32af83791d89b103c9c94c9